src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreWorker.cs

```diff
@@ -432,6 +432,11 @@ public async Task<bool> RestoreAsync(SolutionRestoreRequest request, Cancellatio
             }
         }
 
+        private async Task<bool> IsBulkRestoreCoordinationEnabledAsync()
+        {
+            return await _nugetFeatureFlagService.Value.IsFeatureEnabledAsync(NuGetFeatureFlagConstants.BulkRestoreCoordination);
+        }
```

Comment:
Can it be a one-liner?

```suggestion
        private async Task<bool> IsBulkRestoreCoordinationEnabledAsync() => await _nugetFeatureFlagService.Value.IsFeatureEnabledAsync(NuGetFeatureFlagConstants.BulkRestoreCoordination);
```

Reply:
Given the need for reapproval to changes + the fact that it's syntactic sugar, I'll skip this for now.

I don't lean one or the other way personally, as long as the file itself is consistent.